src/Microsoft.ML.SamplesUtils/ConsoleUtils.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.ML.Data;
```

Comment:
This class is similar to what @shahab is also working on. Both PR are in review. We will have to merge whoever go first...:)

Reply:
Hey I also have dibs on this file! [https://github.com/dotnet/machinelearning/pull/2497/files#diff-7e03e8277aad61a971291e7b1c0ccdd1](https://github.com/dotnet/machinelearning/pull/2497/files#diff-7e03e8277aad61a971291e7b1c0ccdd1)

And none of us have proper comments for public methods :D

---
In reply to: [255763079](https://github.com/dotnet/machinelearning/pull/2476#discussion_r255763079) [](ancestors = 255763079)